src/schedule.ts

```diff
@@ -7,6 +7,7 @@ const main = async () => {
   let errorMessages: (
     | {
         file: string;
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
You should delete this commnet

Reply:
https://typescript-eslint.io/rules/no-explicit-any/